pom.xml

```diff
@@ -526,6 +528,7 @@
               <exclude>**/target/**</exclude>
               <exclude>**/generated-sources/**</exclude>
               <exclude>.github/**</exclude>
+              <exclude>**/banner.txt</exclude>
```

Comment:
this causes compliance issue with release, which requires source files having license header

Reply:
hot fix in https://github.com/apache/hudi/pull/6865